create-cert-bundle.py

```diff
@@ -0,0 +1,50 @@
+import os
+import logging
+import sys
+
+logger = logging.getLogger()
+handler = logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+proxy_url = os.environ.get("ALL_PROXY")
+if proxy_url is None:
+    # No proxy configured
+    sys.exit(0)
+
+# Just try to check if proxy scheme is http or https
+# This does not validate the full url format
+scheme = proxy_url.split(":")[0]
+if scheme != "https":
+    # Not https proxy
+    sys.exit(0)
+
+logger.info("Creating certificate bundle with proxy root CA, and installing dependencies")
+try:
+    import certifi
+
+except ImportError:
+    try:
+        from pip._vendor import certifi
+    except Exception as e:
+        logger.exception("Error creating certificate bundle with proxy root CA")
+        sys.exit(1)
+
+p1 = certifi.where()
+p2 = os.environ.get("GG_ROOT_CA_PATH")
+p3 = './ca-bundle.crt'
+try:
+    with open(p1, 'r') as f1, open(p2, 'r') as f2, open(p3, 'w') as f3:
+        f3.write(f1.read())
+        f3.write(f2.read())
+except Exception:
+    logger.exception("Error creating certificate bundle with proxy root CA")
+    sys.exit(1)
+try:
+    from pip import main as pip
+
+    pip(['install', '--cert', './ca-bundle.crt', 'boto3', 'awsiotsdk', 'urllib3==1.26.9', '--user'])
```

Comment:
can you add a comment about why you need urllib3==1.26.9?

Reply:
Sure, I'll add that.

Reply:
Actually, minimum is 1.26.7 that's needed for this to work as it includes a bug fix with https hostname verification. (https://github.com/urllib3/urllib3/releases/tag/1.26.7)
Does it make sense to not pin it to 1.26.7 and let it install latest, one problem with that is if the user already has that installed with an older version, latest won't actually be installed and they'll be left with a non working version.

Reply:
You can just use `>=`

Reply:
Ohh, yes, don't know why did I miss that. Thanks!